project/scalastyle_config.xml

```diff
@@ -85,11 +85,6 @@
             <parameter name="maxTypes"><![CDATA[30]]></parameter>
         </parameters>
     </check>
-    <check level="error" class="org.scalastyle.scalariform.CyclomaticComplexityChecker" enabled="true">
```

Comment:
We've discussed how handy this rule is multiple times. I'd remove it if there are no objections.

Reply:
I can move this into a separate PR and disable in Java too.